### PR TITLE
Update submissions-teams.md

### PR DESCRIPTION
**Pull request by dhairyya**
Updated default behavior for new tenants vs existing tenants

### CODE REVIEW

**Comment by dhairyya** [comment]
@chrisda could you please approve this?

File: microsoft-365/security/office-365-security/submissions-teams.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ User reporting of messages in Teams is made of two separate settings:
 
 - **In the Teams admin center**: On by default and controls whether users are able to report messages from Teams. When this setting is turned off, users can't report messages within Teams, so the corresponding setting in the Microsoft 365 Defender portal is irrelevant.
 
-- **In the Microsoft 365 Defender portal**: On by default. If user reporting of messages is turned on in the Teams admin center, it also needs to be turned on the Defender portal for user reported messages to show up correctly.
+- **In the Microsoft 365 Defender portal**: On by default for new tenants. For existing tenants, they need to enable it. If user reporting of messages is turned on in the Teams admin center, it also needs to be turned on the Defender portal for user reported messages to show up correctly.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-teams.md]:
```suggestion
- **In the Microsoft 365 Defender portal**: On by default for new tenants. Existing tenants need to enable it. If user reporting of messages is turned on in the Teams admin center, it also needs to be turned on the Defender portal for user reported messages to show up correctly.
```

File: microsoft-365/security/office-365-security/submissions-teams.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ The value of this setting is meaningful only if message reporting is turned on i
 
 2. On the **User reported settings** page, go to the **Microsoft Teams** section for the **Monitor reported messages in Microsoft Teams** setting.
 
-   As previously described, this setting is turned on by default, and you typically leave it turned on if message reporting is also turned on in Teams admin center.
+   As previously described, this setting is turned on by default for new tenants, and you typically leave it turned on if message reporting is also turned on in Teams admin center. For existing tenants you need to enable it.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-teams.md]:
```suggestion
   As previously described, this setting is turned on by default for new tenants, and you typically leave it turned on if message reporting is also turned on in Teams admin center. You need to enable it for existing tenants.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dhairyya, kindly commit the suggested changes. Many thanks.

**Pull request merged**